tests/Aspire.StackExchange.Redis.Tests/RedisContainerFixture.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Components.Common;
+using Aspire.Components.Common.Tests;
+using Testcontainers.Redis;
+using Xunit;
+
+namespace Aspire.StackExchange.Redis.Tests;
+
+public sealed class RedisContainerFixture : IAsyncLifetime
+{
+    public RedisContainer? Container { get; private set; }
+
+    public string GetConnectionString() => Container?.GetConnectionString() ??
+        throw new InvalidOperationException("The test container was not initialized.");
+
+    public async Task InitializeAsync()
+    {
+        if (RequiresDockerTheoryAttribute.IsSupported)
+        {
+            Container = new RedisBuilder()
```

Comment:
Why wouldn't we use Aspire and its incoming test framework here instead of Test Containers?

Reply:
It's too heavyweight for our needs.